### PR TITLE
Feature/initial branch

### PR DESCRIPTION
**Pull request by adri-travelperk**
TKU - Recipe app

### CODE REVIEW

File: project/recipes/models.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import uuid
+from django.db import models
+
+
+class Recipe(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=100)
+    description = models.CharField(max_length=100)
```
Comment (volker-travelperk) [inline project/recipes/models.py]: 100 chars? That must be a fairly quick recipe :sweat_smile:

File: project/recipes/models.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import uuid
+from django.db import models
+
+
+class Recipe(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```
Comment (volker-travelperk) [inline project/recipes/models.py:6]: Why do you use a UUID field?

File: project/recipes/models.py (hunk 1 above)
Comment (adri-travelperk) [reply project/recipes/models.py]: Hahah, fair.
Pushing up to 3k chars :)

File: project/recipes/models.py (hunk 2 above)
Comment (adri-travelperk) [reply project/recipes/models.py:6]: No hard motive. I'm just used to them for indexing in the past and I believe using them was good practice in general.
If using integers as id is a hard requirement, I can change it back no prob.

**Review by volker-travelperk — APPROVED** [review]
Good job :muscle:

**Pull request merged**